update.sh

```diff
@@ -93,8 +93,10 @@ if [[ "$skipsetup" == "n" ]] || [[ "$skipsetup" == "N" ]]; then
 	echo $(tput sgr0)
 	echo
     
+	
 	while [[ "$proceed" == "N" ]] || [[ "$proceed" == "n" ]]; do
 		### wipe arrays clean then set the header for each array ###
+		re="\/.*"
```

Comment:
If you did not ad this. Than I think it's a code I was testing with. So it can be removed